mujoco_ros_control/src/mujoco_ros_control_plugin.cpp

```diff
@@ -173,10 +173,10 @@ void MujocoRosControlPlugin::controlCallback(MujocoSim::mjModelPtr /*model*/, Mu
 		controller_manager_->update(sim_time_ros, sim_period, reset_ctrls);
 	}
 
-	if (last_update_sim_time_ros_.toNSec() > 0.0 && (sim_time_ros - last_write_sim_time_ros_).toNSec() > 0.0)
+	if (!last_update_sim_time_ros_.isZero() && (sim_time_ros > last_write_sim_time_ros_)) {
 		robot_hw_sim_->writeSim(sim_time_ros, sim_time_ros - last_write_sim_time_ros_);
```

Comment:
Is there a reason to condition this call? These (similar) conditions should always hold:
- `last_update_sim_time_ros_ >= 0`
- `sim_tim_ros >= last_write_sim_time_ros_`

Reply:
This was taken over from [gazebo_ros_control](https://github.com/ubi-agni/mujoco_ros_pkgs/blob/59bc2590799933e84ea45deb2ee58d9a4fa52814/mujoco_ros_control/src/mujoco_ros_control_plugin.cpp#L175).

I think this could be condensed into `if (sim_time_ros > last_write_sim_time_ros_)` because I think the reason to condition the call to `writeSim` is to avoid passing a duration of 0 to it.